source/agora/test/Base.d

```diff
@@ -1644,7 +1647,8 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
             en.value,
             validator_addresses[en.index],
             enrolled_addresses.filter!(  // don't connect the validator to itself
-                addr => addr != validator_addresses[en.index]).array));
+                addr => addr != validator_addresses[en.index]).array,
+            test_conf.recurring_enrollment));
```

Comment:
You don't need to pass this argument to `makeValidatorConfig`, that function can access `test_conf` directly.